styles/globals.css

```diff
@@ -29,6 +29,6 @@ a {
 @media screen and (max-width: 600px) {
   :root {
     --header-height: 56px;
-    --footer-height: 127px;
```

Comment:
Height of footer for mobile is 384 but not 167
![image](https://user-images.githubusercontent.com/7271329/174514120-889338c9-f94b-4d3c-ae4b-f5ca3bffb70e.png)

Reply:
> Height of footer for mobile is 384 but not 167 ![image](https://user-images.githubusercontent.com/7271329/174514120-889338c9-f94b-4d3c-ae4b-f5ca3bffb70e.png)

if we use 384 it will be very high, maybe need to comfirm with @Kirl70 , is the mobile design dimensions shown in figma correct? are they what we actually need to use?

Reply:
页脚高度是正确的，因为里面包含logo、外链和版权信息，其中外链是可以点击的，需要留足够的空间利于用户点击。